app/routes/account/billing-info/payment-info.js

```diff
@@ -1,4 +1,12 @@
 import Route from '@ember/routing/route';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 
-export default Route.extend({
+export default Route.extend(AuthenticatedRouteMixin, {
+  titleToken() {
+    return this.l10n.t('Payment Info');
+  },
+
+  async model() {
+    return await this.get('authManager.currentUser');
```

Comment:
Why even return this here ? use `authManager.currentUser` directly in the controller/template